backend/src/captcha/captcha-service.ts

```diff
@@ -1,13 +1,14 @@
 import { HttpService } from '@nestjs/axios';
 import { Injectable } from '@nestjs/common';
+import { BadRequestException } from '@nestjs/common';
 
 @Injectable()
 export class CaptchaService {
   VERIFY_URL = 'https://hcaptcha.com/siteverify';
 
   constructor(private readonly httpService: HttpService) {}
 
-  async verifyToken(token: string, ip?: string): Promise<boolean> {
+  async verifyToken(token: string, ip?: string): Promise<any> {
```

Comment:
Don't recall actually touching this, not sure how it changed :/

Reply:
Will need to update this to get rid of the `any`. Should go back to `boolean`